config.m4

```diff
@@ -48,7 +51,7 @@ if test "$PHP_FIBER" != "no"; then
     [fiber_asm_file_prefix="combined_sysv"]
   )
 
-  if test "$fiber_os" = 'linux'; then
+  if test "$fiber_os" = 'linux' || test "$fiber_os" = 'bsd'; then
```

Comment:
This change and the matches on `freebsd*` and `openbsd*` appear unnecessary, as `fiber_os` should have been set to `"linux"` for these systems anyway. Perhaps the tag should be `"posix"` for clarity